src/main/scala/org/bytedeco/sbt/javacpp/Plugin.scala

```diff
@@ -59,16 +59,16 @@ object Plugin extends AutoPlugin {
    *
    * @param version eg. "1.4.2"
    */
-  private def buildPresetVersion(version: String): String =
+  private def buildPresetVersion(version: String): (String, String) =
     version match {
-      case VersionSplit(a :: b :: _) if a < 2 & b < 4 => s"$a.$b"
-      case VersionSplit(_) => version
-      case _ => throw new IllegalArgumentException("Version format not recognized")
+      case VersionSplit(a :: b :: _) if a == 0 || (a == 1 && b <= 3) => (s"$a.$b", "org.bytedeco.javacpp-presets")
+      case VersionSplit(1 :: 4 :: _) => (version, "org.bytedeco.javacpp-presets")
+      case _ => (version, "org.bytedeco")
```

Comment:
Actually just spotted this default match. Are we sure this makes sense? Would it not make sense to just throw on unparseable version?

Reply:
I want to support ```1.5.4-SNAPSHOT```. ```1.5.4-SNAPSHOT``` goes to ```case _```.
Also 1.5.3 goes to ```case _``` too.

Reply:
Fair enough.